content/blog/2020/01/2020-01-06-happy-new-year.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Happy New Year! 2019/2020 edition"
+tags:
+- community
+- new-year-blogpost
+- governance
+author: oleg_nenashev
+---
+
+Jenkins project congratulates all users and contributors with the New Year!
+Let's take a look at some changes this year.
+
+image:/images/post-images/2017-12-31-new-year/card.png[NewYear, role=center]
+
+## Highlights
+
+* Jenkins project link:/blog/2019/03/12/cdf-launch/[joined the Continuous Delivery Foundation (CDF)]
+* We had first ever link:/blog/2019/12/16/board-election-results/[Governance Board and Officer elections]
+* We released link:/blog/2019/03/11/let-s-celebrate-java-11-support/[Java 11 support in Jenkins]
+* link:https://jenkins-x.io/[Jenkins X] has graduated as a Jenkins sub-project and became a new project under umbrella of CDF
+* We successfully participated in outreach programs: link:/projects/gsoc/2019/[Google Summer of Code], link:/events/hacktoberfest/[Hacktoberfest] and link:/blog/2019/09/23/outreachy-audit-log-release/[Outreachy]
+* In October 2019 we reached the record high number of contributions: **915** unique contributors, **124** of them were first-timers (thanks to Hacktoberfest!)
+* We started new special interest groups for link:/sigs/docs/[Documentation] and link:https://groups.google.com/forum/#!forum/jenkinsci-ux[User Experience].
+* A new position of the Documentation officer was also introduced to highlight an important role of documentation in the project
+
+## Project updates
+
+Highlights above do not cover all advancements we had in the project.
+Below you can find slides from the link:https://www.meetup.com/jenkinsmeetup/events/264795368/[Jenkins contributor summit] in Lisbon.
+There we had project updates by officers, SIG and sub-project leaders.
+See the slide deck to know about: Jenkins Core, Jenkins Pipeline, Configuration-as-Code, Security, UX Overhaul, Jenkins Infrastructure, platform support and documentation.
+
+++++
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vTeUXM75UR8m_v5FbldYYNXeVW0CtXkpRydMRvQjBuew2vRyC4cjgLMSUEVNoonfYbKDSbOvasHPpxg/embed?start=false&loop=false&delayms=60000" width="100%" height="600px"></iframe>
+++++
+
+
+## Some stats and numbers
+
+**Community**.
+Over the past year we had **5433** contributors in GitHub repositories (committers, reviewers, issue submitters, etc.).
+We had **1892** unique committers who created **7122** pull requests and **45484** commits, bots excluded.
+Contributors represent **273** companies and **111** countries, 8% of contributors are recognized as independent.
```

Comment:
Should we highlight top companies and contributors here? I can also add doc and infra contributorsthough they listed below